src/main/resources/application.properties

```diff
@@ -11,3 +11,8 @@ spring.jpa.properties.hibernate.format_sql=true
 spring.autoconfigure.exclude[0]=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
 
 #spring.liquibase.drop-first=true
+server.port=9000
```

Comment:
?

Reply:
Same. Application.properties should be in .gitignore